content/2020/11-09-simplified-app-creation-flow.md

```diff
@@ -0,0 +1,35 @@
+---
+applied_at: "2020-10-27"
```

Comment:
My apologies, there was a recent change to the system and I provided a wrong date for you to have it show up correctly. Can you actually change this to 2020-11-09 to reflect the intended Monday launch please?